ci/src/main/java/ci/Email.java

```diff
@@ -0,0 +1,108 @@
+package ci;
+
+import java.util.Properties;
+
+import javax.mail.*;
+import javax.mail.internet.*;
```

Comment:
javax.mail.* should include javax.mail.internet.*, so the second import statement is redundant

Reply:
some functions can't be resovled if I do not import mail.internet. E.g.:
![image](https://user-images.githubusercontent.com/89870851/153494366-bf358e32-5ded-49d3-b683-90f127e153f7.png)